src/chocolatey.tests.integration/scenarios/ListScenarios.cs

```diff
@@ -379,7 +62,7 @@ public override void Because()
             [Fact]
             public void should_contain_packages_and_versions_with_a_space_between_them()
             {
-                MockLogger.contains_message("upgradepackage 1.0.0").ShouldBeTrue();
+                MockLogger.contains_message("upgradepackage 1.0.0").ShouldBeTrue(userMessage: "Warnings: " + string.Join("\n", MockLogger.Messages["Info"]));

```

Comment:
Is these are warnings should it be `MockLogger.Messages["Warning"]`? (or whatever the warning key is...)

Also: Would it make more sense to join with `Environment::NewLine` instead of `"\n"`?

Reply:
The method looks through all the log types, but if we're looking specifically for warnings we can probably avoid needing to do that by telling it where to look.